docs/UML/all.puml

```diff
@@ -65,4 +65,54 @@ StaffManager <--* StaffController
 Controller <|--- StaffController
 Controller ..> InputParser
 StaffManager --> "0..*" Staff : staffs
+
```

Comment:
i think would be better to put all classes first, then all relations at the bottom, easier to see.

Reply:
ahhh didn't pay attention to that. will be modifying soon